run.py

```diff
@@ -0,0 +1,65 @@
+import os
+from src.model import MLProphet
+from src.data_extract import yahoo
+from sklearn.metrics import r2_score
+from datetime import datetime
+import datetime as dt
+
+
+TICKER = os.environ.get("TICKER", "tsla")
+LABEL = os.environ.get("LABEL", "Open")
+# CURRENT_DATE = os.environ.get("CURRENT_DATE", datetime.today().strftime("%Y-%m-%d"))
+CURRENT_DATE = "2022-04-01"
```

Comment:
should the current_date here be os.environ.get("CURRENT_DATE", datetime.today().strftime("%Y-%m-%d")), or the date that user can input